### PR TITLE
LPS-202228 Change Configuration view for Blogs

### PR DESCRIPTION
**Pull request by BarbaraCabrera**
## Motivation

Hi, @liferay-lima ! 👋🏼 

Continuing the epic of [**Change "Configuration" view for all the apps inside Content & Data**](https://liferay.atlassian.net/browse/LPS-197692) I am sending this pr to change the view for the Blogs configuration. Let me know what do you think. Thanks in advance! 😄 

[Link to ticket](https://liferay.atlassian.net/browse/LPS-202228)

## Changes

Change the configuration for Blogs because it should be open in a new page instead of a modal.

## Test Scenario

1. Go to Blogs
2. Click on the kebab menu in the top right
3. Navigate to the sections to check the behaviour and styles

https://github.com/liferay-lima/liferay-portal/assets/93203423/f3adbd20-00b5-4c2d-b9fb-fc5fdaa39db3

### CODE REVIEW

File: modules/apps/blogs/blogs-web/src/main/java/com/liferay/blogs/web/internal/display/context/BlogsConfigurationDisplayContext.java [hunk 1]
```diff
@@ -63,12 +63,8 @@ public String getNavigation() {
...
 
 		_navigation = ParamUtil.getString(
-			_renderRequest, "navigation", "email-from");
-
-		if (PortalUtil.isRSSFeedsEnabled()) {
-			_navigation = ParamUtil.getString(
-				_renderRequest, "navigation", "rss");
-		}
+			_renderRequest, "navigation",
+			PortalUtil.isRSSFeedsEnabled() ? "rss" : "email-from");
```
Comment (AliciaGarciaGarcia) [inline modules/apps/blogs/blogs-web/src/main/java/com/liferay/blogs/web/internal/display/context/BlogsConfigurationDisplayContext.java:67]: We are allowed (now) to use ternary on Java?

**Comment by AliciaGarciaGarcia** [comment]
LGTM
The SF question and waiting for the Front review.

**Comment by jkappler** [comment]
Hi @AliciaGarciaGarcia, please see for example:

https://github.com/liferay/liferay-portal/blob/master/modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/display/context/SelectAccountUsersManagementToolbarDisplayContext.java#L118

https://github.com/liferay/liferay-portal/blob/master/modules/apps/asset/asset-categories-admin-web/src/main/java/com/liferay/asset/categories/admin/web/internal/display/context/AssetCategoriesDisplayContext.java#L1038

https://github.com/liferay/liferay-portal/blob/master/modules/apps/blogs/blogs-web/src/main/java/com/liferay/blogs/web/internal/display/context/BlogsManagementToolbarDisplayContext.java#L194

 https://github.com/liferay/liferay-portal/blob/master/modules/apps/calendar/calendar-web/src/main/java/com/liferay/calendar/web/internal/display/context/CalendarDisplayContext.java#L535

We have around 62 usages in display contexts.....

Hope this helps

**Comment by AliciaGarciaGarcia** [comment]
I was more concerned because during a time we were not allowed to use it (I do not why)
perfect for me!!! <3

**Comment by jkappler** [comment]
We removed the code since it's code from Lima Team as @BarbaraCabrera mentioned

**Review by ambrinchaudhary — APPROVED** [review]

**Comment by ambrinchaudhary** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward:force

**Comment by ealonso** [comment]
ci:forward:force

**Pull request closed without merging**